composeApp/src/commonMain/kotlin/com/github/familyvault/models/enums/chat/ChatMessageContentType.kt

```diff
@@ -3,5 +3,5 @@ package com.github.familyvault.models.enums.chat
 enum class ChatMessageContentType {
     TEXT,
     VOICE,
-    IMAGE
+    MEDIA
```

Comment:
Po przejrzeniu kodu powinno zostać `IMAGE` ponieważ renderowanie jest stricte nastawione na to